zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/usertask/UserTaskProcessor.java

```diff
@@ -43,6 +43,9 @@ public class UserTaskProcessor implements TypedRecordProcessor<UserTaskRecord> {
   private static final String USER_TASK_COMPLETION_REJECTION =
       "Completion of the User Task with key '%d' was rejected by Task Listener";
 
+  private static final String USER_TASK_ASSIGNMENT_REJECTION =
+      "Assignment of the User Task with key '%d' was rejected by Task Listener";
```

Comment:
🔧 I see now that we could improve the wording here for both rejection messages:

```suggestion
  private static final String USER_TASK_ASSIGNMENT_REJECTION =
      "Assignment of the User Task with key '%d' was denied by Task Listener";
```

Reply:
Great suggestion, thank you @korthout 👍